README.md

```diff
@@ -25,6 +25,8 @@
 
 ## Overview
 
+**Note:** Documents in this repo are in pre-release version. For the released version, you can refer to [IBM Knowledge Center](https://www.ibm.com/support/knowledgecenter/SSHKN6/installer/1.x.x/index.html)
+
 Operand Deployment Lifecycle Manager is used to manage the lifecycle of a group of operands. Checkout design document at [here](./docs/design/operand-deployment-lifecycle-manager.md).
```

Comment:
Check the design document [here](./docs/design/operand-deployment-lifecycle-manager.md).